### DISCUSSION TITLE
이정도 애니메이션 과한가요??

### DISCUSSION THREAD

**Discussion by jjun990908** [softeerbootcamp-2nd/A5-Idle#295] [Q&A] [answered]
![ezgif com-video-to-gif (5)](https://github.com/softeerbootcamp-2nd/A5-Idle/assets/39405559/f9ca1831-15ca-43fb-84c9-b7e841c8e6c7)

위에 차량은 괜찮아서 push 한 상태인데

아래 리스트 날아오는 애니메이션은 과한가요~~??

**Comment by Wongilk** [comment, marked as answer]
아 공유하기 때문에 안되겠네요 아쉽다
